### PR TITLE
Remove global commands

### PR DESCRIPTION
**Pull request by stefanmcshane**
- Remove all commands from being declared at a global level
- Initialise config once, then pass into each command
- Create `cli/cmd/commands` folder for registering commands

### CODE REVIEW

File: cli/cmd/preview/v2beta1/default_driver.go [hunk 1]
```diff
@@ -0,0 +1,375 @@
+package v2beta1
```
[the remark this answers is not in the document]
Comment (stefanmcshane) [reply cli/cmd/preview/v2beta1/default_driver.go:1]: I didnt change this. Let me reset to master and see if this disappears.

[the remark this answers is not in the document]
Comment (stefanmcshane) [reply cli/cmd/preview/v2beta1/default_driver.go]: Fixed. Resolving this now

**Review by jose-fully-ported — APPROVED** [review]

File: cli/cmd/commands/auth.go [hunk 2]
```diff
@@ -70,101 +70,104 @@ func init() {
...
+			return fmt.Errorf("error creating porter API client: %w", err)
+		}
 	}
 
 	user, err := client.AuthCheck(ctx)
+	if err != nil {
+		if !strings.Contains(err.Error(), "Forbidden") {
+			return fmt.Errorf("unexpected error performing authorization check")
+		}
+		fmt.Println(err)
```
Comment (ianedwards) [inline cli/cmd/commands/auth.go:93]: Is this print expected?

**Review by ianedwards — APPROVED** [review]

File: cli/cmd/commands/auth.go (hunk 2 above)
Comment (stefanmcshane) [reply cli/cmd/commands/auth.go:93]: Nope. @sdess09 is going to remove it in a followup PR

**Pull request merged**